shell/browser/javascript_environment.cc

```diff
@@ -360,10 +360,13 @@ void JavascriptEnvironment::OnMessageLoopDestroying() {
   base::CurrentThread::Get()->RemoveTaskObserver(microtasks_runner_.get());
 }
 
-NodeEnvironment::NodeEnvironment(node::Environment* env) : env_(env) {}
+NodeEnvironment::NodeEnvironment(node::Environment* env,
+                                 node::IsolateData* isolate_data)
+    : env_(env), isolate_data_(isolate_data) {}
 
 NodeEnvironment::~NodeEnvironment() {
   node::FreeEnvironment(env_);
```

Comment:
Can this instead be

```
auto* isolate_data = env_->isolate_data();
node::FreeEnvironment(env_);
node::FreeIsolateData(isolate_data);
```

Avoids having to change the class definition.

Reply:
Sure, done!